src/components/Intro/Intro.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useRef } from 'react';
+import {
+  m, useReducedMotion, useScroll, useTransform, useWillChange, useSpring, SpringOptions,
+} from 'framer-motion';
+import { Bracket } from '../Bracket';
+import { FlexBox } from '../FlexBox';
+
+export const Intro = () => {
+  const prefersReduceMotion = useReducedMotion();
+  const willChange = useWillChange();
```

Comment:
This is new - we need the CSS will change property to minimize the visual artifacts in the brackets due to the user of transform (translating x position). This hook is provided by Framer Motion and is supposed to improve the performance of using will change